### ISSUE TITLE
[FlexibleColumnLayout] : Screen Reader announces it as 3 regions even when you use MidColumnFullScreen layout

### ISSUE THREAD

**Issue by achuth2912** [SAP/ui5-webcomponents#9306]
### Bug Description

FlexibleColumnLayout is always announcing to have 3 regions when ever we use MidColumnFullScreen layout.
We have a usage of Custom layouts configuration getLayoutsByMedia() to set the width of the layout.
Probably using this exposed Custom layouts configuration getLayoutsByMedia() is causing the issue with Screen readers to read it as 3 regions even when we have only one layout on the UI.

### Affected Component

FlexibleColumnLayout

### Expected Behaviour

When we have only one layout on the UI even the Screen reader must announce 1 region instead of 3 regions.
The aria-hidden="true" is gone missing while using the Custom layouts configuration getLayoutsByMedia() to set the width of the layout.

### Isolated Example

https://codesandbox.io/p/devbox/flexlayoutaccessibility-issue-t9kqf5

### Steps to Reproduce

1. Do a inspect element with using the Custom layouts configuration getLayoutsByMedia() to set the width of the layout.
2. Attaching 2 screenshots one with out having the Custom layouts configuration and the other one having the Custom layouts configuration set.
3.
...

### Log Output, Stack Trace or Screenshots

<img width="1728" alt="Screenshot 2024-06-26 at 7 02 04 PM" src="https://github.com/SAP/ui5-webcomponents/assets/71249041/4916a0e1-4745-483d-852f-6dbacee58397">
<img width="1728" alt="Screenshot 2024-06-26 at 7 02 16 PM" src="https://github.com/SAP/ui5-webcomponents/assets/71249041/c2cf668c-f80e-4bba-99b3-50575ea15ec1">

### Priority

None

### UI5 Web Components Version

1.24.5

### Browser

Chrome

### Operating System

Mac OS

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by kineticjs** [comment]
Hi @achuth2912 ,

Thank you for the sample and the initial analysis. In my testing the issue happens when the width of the hidden column is given as `"0"` in your custom configuration, e.g.:
`[ "0",  "100%",  "0",  ];`
When I replaced `"0"` with `"0px"`, e.g.
`[ "0px",  "100%",  "0px",  ];`
the issue no longer reproduces on my side.

Can you please edit and retest on your side?

Thanks

**Comment by achuth2912** [comment]
Hi @kineticjs 

Thanks for your valuable inputs here.
After adding replacing  `"0"` with `"0px"` its working as expected.

Thanks
